Dockerfile

```diff
@@ -480,7 +480,8 @@ RUN pip install flashtext && \
     pip install chainercv && \
     pip install plotly_express && \
     pip install albumentations && \
-    pip install allennlp && \
+    # TODO: install deps when underlying dependency is fixed. https://github.com/Kaggle/docker-python/issues/548
+    pip install --no-dependencies allennlp && \
```

Comment:
Make sure all the useful dependencies are satisfied and add any missing ones:
https://github.com/allenai/allennlp/blob/master/requirements.txt

You can skip the test, docs and AWS related ones.

Reply:
done